core/domain/exp_jobs_one_off.py

```diff
@@ -492,23 +491,17 @@ def map(item):
         # invalid parent-child relations that we find.
         err_dict = {}
 
-        htmlValidation_filepath = os.path.join(
-            feconf.CONTENT_VALIDATION_DIR, 'htmlValidation.yaml')
-        htmlValidation_yaml = utils.get_file_contents(htmlValidation_filepath)
-        # Convert the yaml content to dictionary.
-        htmlValidation_dict = utils.dict_from_yaml(htmlValidation_yaml)
-
-        allowed_parent_list = htmlValidation_dict['allowed_parent_list']
-        allowed_tag_list = htmlValidation_dict['allowed_tag_list']
+        allowed_parent_list = feconf.ALLOWED_PARENT_LIST
```

Comment:
Hi @AllanYangZhou @seanlip 
I have changed the location of these constants here as suggested by Sean. If you think the `yaml` approach is fine, I'll revert these changes.
Thanks

Reply:
I think constants here are fine. That said, consider making them class-level constants here, rather than exposing them generally by placing them in feconf.